x264/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+
+PREFIX_X264="${PREFIX_PROJECT}/phoenix-rtos-ports/x264"
+PREFIX_X264_BUILD="${PREFIX_BUILD}/x264"
+PREFIX_X264_CONFIG="${PREFIX_X264}/patches"
+PREFIX_X264_MARKERS="${PREFIX_X264_BUILD}/markers"
+
+PKG_NAME="x264-master.tar.bz2"
+PKG_URL="https://code.videolan.org/videolan/x264/-/archive/master"
```

Comment:
don't use HEAD of the repo, always use direct commit reference for reproducible builds (eg. `https://code.videolan.org/videolan/x264/-/archive/31e19f92f00c7003fa115047ce50978bc98c3a0d/x264-31e19f92f00c7003fa115047ce50978bc98c3a0d.tar.gz` - the latest commit from the `stable` branch)

Reply:
Thanks, missed this. Added URL to direct commit.